cd2h_repo_project/modules/frontpage/templates/cd2hrepo_frontpage/index.html

```diff
@@ -30,16 +30,27 @@
+      <b>InvenioRDM</b> enables organizations to securely house research products and <b>make them discoverable, shareable, and citable</b> -- from publications and presentations to datasets, software, lay summaries, policy documents, and guidelines. <b>InvenioRDM</b> is being developed as part of a large, multi-organization collaboration in partnership with the <a href="https://home.cern/">European Organization for Nuclear Research (CERN)</a>, birthplace of the World Wide Web and developers of the <a href="https://zenodo.org/">Zenodo</a> RDM for the European community. Together we are working to build a wide range of features that can help power biomedical research and support data sharing, innovation, knowledge dissemination, and interdisciplinary collaboration.
     </p>
 
+    <p>
+      Our work is driven by user needs and informed by best practices and standards, including those that help define <a href="http://ngr.coar-repositories.org/">Next Generation Repositories</a> as a foundation for a distributed, globally-networked infrastructure for scholarly communication, discovery, and innovation.
+    </p>
+    
+    <p>
+      <b>Contact us</b>. <a href="mailto:invenioRDM@northwestern.edu">Email us</a> if you have questions or if you’d like to request a test user name and password for this site so you can create records of your own.
+    <br>
+    <br>
+      <b>Join us</b>. Get updates and test new releases on the <a href="https://invenio-software.org/">product site</a>, <a href="mailto:invenioRDM@northwestern.edu">email us</a> to request a demonstration or chat, take a look at our competitive landscape analysis,  and get involved with collaboration and discussion through our <a href="https://github.com/inveniosoftware/invenio-app-rdm">open source project</a>. 
+    </p>
+    
     <br>
     <p class="footnote">
-      This project is funded in part by the <a href="https://ctsa.ncats.nih.gov/cd2h/">CTSA National Center for Data to Health (CD2H)</a> and supported by the NIH National Center for Advancing Translational Sciences, Grant Number U24TR002306, to help guide the cultural and technological changes necessary for data and informatics to improve research and health care.
+      InvenioRDM is powered by the <a href="https://galter.northwestern.edu/">Galter Health Sciences Library & Learning Center</a> and supported in part by the Feinberg School of Medicine and the NIH National Center for Advancing Translational Sciences (NCATS) through grants U24TR002306 (<a href="https://ctsa.ncats.nih.gov/cd2h/">CD2H</a>) and UL1TR001422 (<a href="https://www.nucats.northwestern.edu/">NUCATS</a>).
```

Comment:
The footer has "Powered by Invenio" already. Which one do we want to keep, how do we want to reword it? Potential synonyms for this sentence are "built by" / "developed by" / "is the effort of"...

Reply:
What about ["Built with", "Built on"] Invenio?

Reply:
Went with "Built on". This page can be previewed on stage now.